WORKSPACE

```diff
@@ -57,3 +57,14 @@ http_archive(
         "https://github.com/bazelbuild/rules_cc/archive/0d5f3f2768c6ca2faca0079a997a97ce22997a0c.zip",
     ],
 )
+
+# Tools for building deb, rpm and tar files.
```

Comment:
Could you please change this comment to "Needed as a transitive dependency of @io_bazel" ?